scripts/nesi_scripts/run_adv_im_obs_maui.sl

```diff
@@ -52,15 +46,15 @@ do
     do
         # check for status
         # skip if completed
-        res=`python $gmsim/workflow/scripts/verify_adv_IM.py $path_event_out $adv_IM_model`; res_return_code=$?
+        python $IMPATH/../Advanced_IM/check_adv_IM_status.py $path_event_out $adv_IM_model; res_return_code=$?
```

Comment:
IMPATH should be changed to be the second level IM_calculation directory so that you don't have to back track here and just below?
The PYTHONPATH export can be adapted to work with the change too

Reply:
thats doable and easy to fix, the reason i did not changed that was to minimize the changes, as its not part of the issue.

Reply:
I'd change it over. Clean code is better than legacy code.